biothings/web/services/metadata.py

```diff
@@ -72,7 +75,7 @@ def __init__(self, indices, client):
         self.client = client
 
         # initial refresh
-        loop = asyncio.get_event_loop()
+        loop = get_loop()
```

Comment:
@sengineer0 this seems an odd fix, because this code is under `biothings.web` and this fix uses a helper `get_loop` from `biothings.hub`.

Reply:
ah, I just forgot that, doing like this will create depending on `biothings.hub`, right?
I will change this fix, to remove that dependency. thank you for noticing me :)
@newgene